spec/board-spec.rb

```diff
@@ -0,0 +1,70 @@
+require '../lib/game.rb'
+
+describe '::Board' do
+  let(:board) {Board.new}
+  let(:show_board) {board.show_board}
```

Comment:
Not needed.

Reply:
Okay, what this test actually does is makes sure that the board has 3 rows. It doesn't actually print. What do you think?